src/dispatcher.rs

```diff
@@ -573,10 +551,10 @@ pub extern "C" fn proxy_on_http_call_response(
 
 #[no_mangle]
 pub extern "C" fn proxy_on_grpc_receive(_context_id: u32, token_id: u32, response_size: usize) {
-    DISPATCHER.with(|dispatcher| dispatcher.on_grpc_receive(token_id, response_size))
+    DISPATCHER.with(|dispatcher| dispatcher.on_grpc_call_response(token_id, 0, response_size))
 }
 
 #[no_mangle]
 pub extern "C" fn proxy_on_grpc_close(_context_id: u32, token_id: u32, status_code: u32) {
-    DISPATCHER.with(|dispatcher| dispatcher.on_grpc_close(token_id, status_code))
+    DISPATCHER.with(|dispatcher| dispatcher.on_grpc_call_response(token_id, status_code, 0))
```

Comment:
I think you need to do this mapping inside dispatcher, you cannot differentiate unary vs stream here.

Reply:
Fixed. Does it mean not to provide `on_grpc_call_response` as dispatcher member?

Reply:
Correct.